buffer/src/main/java/io/netty/buffer/ByteBufProcessor.java

```diff
@@ -21,11 +21,13 @@
 /**
  * @deprecated Use {@link ByteProcessor}.
  */
+@Deprecated
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @Deprecated annotation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)